docker-compose-test.yml

```diff
@@ -1,15 +1,19 @@
 version: "3"
+name: zeno-test
 services:
   frontend:
-    container_name: zeno-frontend
+    container_name: zeno-frontend-test
     build:
       context: ./frontend
-      dockerfile: Dockerfile.prod
+      dockerfile: Dockerfile.dev
+    volumes:
+      - ./frontend:/app
+      - /app/node_modules/
     ports:
       - "5173:5173"
-    command: npm run preview -- --port 5173 --host 0.0.0.0
+    command: ["npm", "run", "dev", "--", "--host", "0.0.0.0"]
```

Comment:
Why dev? I had it do preview because it gets rid of the flakyness. It's flaky because of vite doing HMR stuff I think.